R/slice.R

```diff
@@ -18,14 +22,18 @@
 #' x
 #'
 #' vec_slice(mtcars, 1:3)
-vec_slice <- function(x, i) {
-  .Call(vctrs_slice, x, maybe_missing(i))
+vec_slice <- function(x, i, ...) {
+  return(.Call(vctrs_slice, x, maybe_missing(i), environment()))
```

Comment:
`ellipsis::check_dots_used()`?

Reply:
Should the UI be `on.exit(check_dots_unused())`? I think hiding the `on.exit()` makes it harder to read. Also harder to experiment with. Might also lead to bugs by writing things like:

```r
check_dots_unused()

on.exit(cleanup())
```